app/lib/tariff_synchronizer/mailer.rb

```diff
@@ -58,10 +57,9 @@ def applied(update_names, import_warnings)
       @update_names = update_names
       @import_warnings = import_warnings
       # if 'presence errors' are ignored during tariff update then we can display them in email body
-      if TariffSynchronizer.ignore_presence_errors
+      if TaricSynchronizer.ignore_presence_errors
```

Comment:
Curious, I suspect we can actually delete the presence error feature

Reply:
We've not used it in the three years I've been here

Reply:
I'd prefer to do this in separate ticket